pkg/app/app.go

```diff
@@ -17,6 +18,8 @@ import (
 
 const (
 	WriteReadReadPerms = 0o644
+
+	runDir = "runs"
```

Comment:
probably shall go to constants.go (same as bin, and any other dirname for the tree under .avalanche-cli)

Reply:
done